scenariorun/src/main/kotlin/com/cosmotech/scenariorun/ContainerFactory.kt

```diff
@@ -1037,15 +1039,34 @@ internal class ContainerFactory(
           // PROD-8074: In the AMQP Consumers, Shared Access Policy credentials take precedence over
           // the tenant client credentials => we can therefore safely append the former to the
           // existing env vars.
+          // PROD-8599: AZURE_EVENT_HUB_SHARED_ACCESS_POLICY_ENV_VAR,
+          // AZURE_EVENT_HUB_SHARED_ACCESS_KEY_ENV_VAR
+          // are renamed, but are kept for backward compability reason
+          val sharedAccessPolicyNamespaceName =
+              eventBus.authentication.sharedAccessPolicy?.namespace?.name
```

Comment:
Could you also include the `throw` null-safety statements as part of the 2 variables assignments? Thanks.